pkg/agent/datamodel/sig_config.go

```diff
@@ -70,6 +71,11 @@ var AvailableUbuntu1804Distros []Distro = []Distro{
 	AKSUbuntuArm64Containerd1804Gen2,
 }
 
+var AvailableUbuntu2004Distros []Distro = []Distro{
```

Comment:
this wont be consumed here, used in RP. leaving it as a reference for the RP PR, will remove as cleanup